### PR TITLE
feature: ArrayPushFixer now fix short arrays

### PR DESCRIPTION
**Pull request by The-Hasanov**
```php
 array_push($a, [$b, $c]);
 //result
 $a[] = [$b, $c];
```

### CODE REVIEW

File: tests/Fixer/Alias/ArrayPushFixerTest.php [hunk 1]
```diff
@@ -113,6 +113,21 @@ public function provideFixCases(): iterable
...
+        yield 'simple short array' => [
+            '<?php $a[] = [$b];',
+            '<?php array_push($a, [$b]);',
+        ];
+
+        yield 'multiple element short array' => [
+            '<?php $a[] = [[], [], $b, $c];',
+            '<?php array_push($a, [[], [], $b, $c]);',
+        ];
+
```
Comment (SpacePossum) [inline tests/Fixer/Alias/ArrayPushFixerTest.php:130]: can you add:
``php
        yield 'second argument wrapped in `(` `)`' => [
            '<?php $a::$c[] = ($b);',
            '<?php array_push($a::$c, ($b));',
        ];
```

File: src/Fixer/Alias/ArrayPushFixer.php [hunk 2]
```diff
@@ -197,7 +197,9 @@ private function getSecondArgumentEnd(Tokens $tokens, int $index, int $endIndex)
             return null;
         }
 
-        $index = $tokens->getNextMeaningfulToken($index);
+        if (!$tokens[$index]->isGivenKind(CT::T_ARRAY_SQUARE_BRACE_OPEN)) {
```
Comment (SpacePossum) [inline src/Fixer/Alias/ArrayPushFixer.php:200]: >$index = $tokens->getNextMeaningfulToken($index);

removing this seems good to me :+1: 
however I think this `if` block is not needed either

**Comment by SpacePossum** [comment]
@julienfalque can you approve running the CI on this PR? thanks in advance

**Review by SpacePossum — APPROVED** [review]

File: src/Fixer/Alias/ArrayPushFixer.php (hunk 2 above)
Comment (The-Hasanov) [reply src/Fixer/Alias/ArrayPushFixer.php]: @SpacePossum  : I checked your suggestion. Everything looks right.

Comment (SpacePossum) [reply src/Fixer/Alias/ArrayPushFixer.php]: thanks so much for the PR :+1:  RTM from me : )

**Comment by keradus** [comment]
converting to feature.
rule was covering some cases, but not all - now is covering more cases, which is considered a feature

**Comment by keradus** [comment]
kudos!

**Pull request merged**